templates/octavia/config/octavia.conf

```diff
@@ -0,0 +1,110 @@
+[DEFAULT]
+debug=True
+# debug={{.Debug}}
+# transport_url={{.RabbitTransportURL}}
+rpc_response_timeout=60
+log_file=/var/log/octavia/octavia.log
+log_dir=/var/log/octavia
+[api_settings]
+bind_host=192.168.1.147
+bind_port=9876
+auth_strategy=keystone
+# enabled_provider_drivers=amphora: The Octavia Amphora driver.,octavia: Deprecated alias of the Octavia Amphora driver.
+healthcheck_enabled=True
+tls_cipher_prohibit_list=
+default_listener_tls_versions=TLSv1.2,TLSv1.3
+default_pool_tls_versions=TLSv1.2,TLSv1.3
+[database]
+[health_manager]
+health_update_threads=4
```

Comment:
This seems very low, are you sure this is the right value?
This is an important value for the scalability of the number of amphora instances the deployment can handle.

Reply:
Good point. I'll revise all of the configs when adding those services